src/coldfront_plugin_api/scim_v2/auth_middleware.py

```diff
@@ -0,0 +1,17 @@
+import os
+import logging
+
+from django_scim.middleware import SCIMAuthCheckMiddleware
+from mozilla_django_oidc.contrib.drf import OIDCAuthentication
+
+logger = logging.getLogger(__name__)
+
+
+class SCIMColdfrontAuthCheckMiddleware(SCIMAuthCheckMiddleware):
+    def process_request(self, request):
+        if not request.user or not request.user.is_staff:
+            # PLUGIN_AUTH_OIDC implies DRF OIDC backend is configured
+            if os.getenv("PLUGIN_AUTH_OIDC") == "True":
+                oidc_auth_obj = OIDCAuthentication()
+                request.user = oidc_auth_obj.authenticate(request)
+            return super().process_request(request)
```

Comment:
Remove this from the conditional and let it always be called. 

All we want this entire function to do is add user to request and let everything flow as usual.